data/transactions/verify/txn_test.go

```diff
@@ -369,6 +371,85 @@ func TestDecodeNil(t *testing.T) {
 	}
 }
 
+func TestTxnGroupWithTracer(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+
+	proto := config.Consensus[protocol.ConsensusCurrentVersion]
+
+	account := keypair()
+	accountAddr := basics.Address(account.SignatureVerifier)
+
+	// `#pragma version 6; int 1;`
+	program1 := []byte{0x06, 0x81, 0x01}
+	program1Addr := basics.Address(logic.HashProgram(program1))
+
+	// `#pragma version 6; byte "test"; pop; int 1;`
```

Comment:
Could you use `logic.AssembleString`? Or is there some circularity going on?

Reply:
I just realized I can explicitly use pushint/pushbytes to avoid any possible ambiguity. Change made in 5927a63c7